nyc_taxis/TEST_PROCEDURES.md

```diff
@@ -0,0 +1,54 @@
+# Test Procedures
+
+## searchable_snapshot 
+
+A test procedure for measuring performance for Searchable Snapshot feature. It runs the same search query as the default test procedure `append-no-conflicts`.
+
+In contrast with `append-no-conflicts` which running query on index with local storage, this test procedure runs query on index backed by a remote snapshot.
+
+The remote snapshot stores in Amazon S3, an Amazon S3 bucket for storing the snapshot and an AWS account user credential that has permission to access the bucket is required for the workload,
+to learn more in configuring Amazon S3 bucket as a snapshot repository, see the [OpenSearch docs](https://opensearch.org/docs/latest/opensearch/snapshots/snapshot-restore#amazon-s3).
```

Comment:
Maybe reword to something like
```
This test procedure requires that a remote snapshot be stored in Amazon S3. To learn more about configuring Amazon S3 as a snapshot repository, see the [OpenSearch docs](https://opensearch.org/docs/latest/opensearch/snapshots/snapshot-restore#amazon-s3)
```

Reply:
Thanks for the advice, I made a modification to the expression to make it clearer.
`The test procedure will create a remote snapshot that stored in Amazon S3, so an Amazon S3 bucket for storing the snapshot and an AWS account user credential that has permission to access the bucket are required to run the test procedure. To learn more about configuring Amazon S3 as a snapshot repository, see the [OpenSearch docs](https://opensearch.org/docs/latest/opensearch/snapshots/snapshot-restore#amazon-s3).`